packages/zowe-explorer/package.json

```
@@ -178,6 +173,11 @@
           "dark": "./resources/dark/history.svg"
         }
       },
+      {
+        "command": "zowe.jobs.tabularView",
```

Should the command be `tableView` instead of `tabularView` to be consistent with the class name?
```suggestion
        "command": "zowe.jobs.tableView",
```